### PR TITLE
docs: fix code example styles to conform with style guide

### PR DESCRIPTION
**Pull request by preethamvishy**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [x] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: #26581 

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/content/guide/universal.md [hunk 1]
```diff
@@ -431,7 +432,7 @@ dist/
 
 To run the app on the server, type the following in a command shell.
 
-<code-example format="." language="bash" linenums="false">
+<code-example language="sh" class="code-shell" linenums="false">
```
Comment (jenniferfell) [inline aio/content/guide/universal.md]:
```
<code-example language="sh" class="code-shell">
```

File: aio/content/guide/universal.md [hunk 2]
```diff
@@ -459,7 +460,7 @@ You can add these in the `"server"` section of the Angular configuration file, `
 
 To run a production build of your app with Universal on your local system, use the following command.
 
-<code-example format="." language="bash" linenums="false">
+<code-example language="sh" class="code-shell" linenums="false">
```
Comment (jenniferfell) [inline aio/content/guide/universal.md]:
```
<code-example language="sh" class="code-shell">
```

**Review by jenniferfell — CHANGES_REQUESTED** [review]
Sorry, I missed the linenum part of these the first time. Doesn't look like we need or should have linenum in these two in universal. Thx.

**Comment by jenniferfell** [comment]
Please also remember to squash your commits. Thanks! (https://github.com/ExpertSupport/doc-workflow/wiki/Squashing-Commits)

**Comment by preethamvishy** [comment]
> Good catch. This looks great. While you're here, could I ask you to make one more change? We got the markup totally wrong on one (oops):
> 
> https://angular.io/guide/universal#step-1-install-dependencies
> 
> The `npm install` command in that section should also be a code example
> 
> ```
> <code-example language="sh" class="code-shell">
>   npm install --save @angular/platform-server @nguniversal/module-map-ngfactory-loader ts-loader
> </code-example>
> ```

Thanks. I've made the required changed and added a few more changes for your review.

**Comment by brandonroberts** [comment]
@preethamvishy will you rebase your branch on master and fix the merge conflcts? Thanks

**Comment by brandonroberts** [comment]
@preethamvishy will you rebase this branch, resolve conflicts, and squash down to a single commit?

**Comment by brandonroberts** [comment]
Reopen if you're still interested in landing these changes

**Pull request closed without merging**